### DISCUSSION TITLE
Mounting the Enclosure--Drill holes?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#722] [REM] [answered]
Before I drill holes in my expensive, otherwise water-tight enclosure I thought I'd ask if this is the way others do it.

I'll be mounting it either on a 4x4 post or on a concrete block wall with concrete anchors.

I can use silicone or similar to make sure the holes are waterproof.

**Comment by rstrouse** [comment]
Which enclosure do you have?

**Reply by davthomaspilot** [reply]
Altelix 14x12x8 Industrial DIN Rail Enclosure Fiberglass NEMA 4X IP66

**Reply by rstrouse** [reply]
If it were me I would put a steel strap between the upper and lower mounts on the enclosure.  This way you do not create penetrations on the back of the enclosure.

**Reply by davthomaspilot** [reply]
The back of the enclosure is flat, so I'm not clear on what you mean.

Maybe I ordered the wrong one, but I was trying to get the one that was recommended.

**Reply by rstrouse** [reply, marked as answer]
There should be 4 screw holes that have an embedded nut for mounting.  One in each of the corners.  I sunk two galvanized pipes into the pad to mount mine.

**Reply by davthomaspilot** [reply]
Looking again at the website:

> Stainless steel brackets are included for secure wall mounting.


![image](https://user-images.githubusercontent.com/6830886/232921769-93c6c01b-9c3d-4668-b980-268bf3ad0f5c.png)

But, I didn't get them.  I'll have to call them tomorrow.

Good idea on the pipes--

I have a copper injection system mounted on a 4x4 post that the pool builder is very enthusiastic about.  Dozens, maybe hundreds of high end pools in our area use it.

It worked great while I used it, but the folks on TFP were very much against that approach and scared me into not using it.  I haven't used it for several years now, so I figure I'll take it down and re-use the mount and wiring for the REM chemical controller.

Thanks!

**Reply by rstrouse** [reply]
The folks at TFP are not wrong on that.  Unfortunately, copper is good in fish ponds but it pretty much stains plaster like crazy especially in areas where the evaporation is high and the water replacement is low.  I think the idea comes from the malachite treatments you do for koi but that is more a lice and fugus killer than a sanitizer.

You can use pipe straps to affix it to the pipes.  This will allow you to move the panel up and down to adjust it into fixed conduit if you are using it.
